torch/_numpy/_getlimits.py

```diff
@@ -0,0 +1,20 @@
+import contextlib
+
+import torch
+
+from . import _dtypes
+
+
+def finfo(dtyp):
+    torch_dtype = _dtypes.dtype(dtyp).torch_dtype
+    return torch.finfo(torch_dtype)
+
+
+def iinfo(dtyp):
+    torch_dtype = _dtypes.dtype(dtyp).torch_dtype
+    return torch.iinfo(torch_dtype)
+
+
+@contextlib.contextmanager
+def errstate(*args, **kwds):
+    yield
```

Comment:
hmm?

Reply:
This one sets different error flags on FP numbers on CPU. I think we should remove it given that we're not implementing it. @ev-br

Reply:
The original is https://numpy.org/doc/stable/reference/generated/numpy.errstate.html; The noop here is/was to minimize churn in numpy code.